react/src/lib/components/DeckGLMap/components/Map.tsx

```diff
@@ -600,7 +620,7 @@ const Map: React.FC<MapProps> = ({
             {scale?.visible ? (
                 <DistanceScale
                     {...scale}
-                    zoom={viewState?.zoom}
+                    zoom={-5}
```

Comment:
This breaks scalebar to adapt to the changing zoom. We can take the zoom from 1st view state to fix this.
If users will use multi-views, they can hide the default scalebar and use the distance scale component to add a separate scalebar for each view.